### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by nopitjahjadi** [arennuit/vrep_ros_control_example#1]
hello,
are you using v-rep version 3.1.3 ? 
well, I'd clone this vrep ros control, and I'm using v 3.1.2 
but I failed to build the lib file from the beginning. the compilation is terminated

**Comment by arennuit** [comment]
@nopitjahjadi yes I am using 3.1.3. Now you need to give me more details on how the compilation terminates and how you compile it... Thanks ;)

**Comment by nopitjahjadi** [comment]
I put the folder on my catkin /src folder, so I could do catkin_make together with other plugin for V-rep. but somehow it is terminated. so I completely missing here. in version 3.1.2 there're no vrep_skeleton_msg_and_srv, but in displayText.h there's a file for vrep_skeleton_msg_and_srv. I can't found it anywhere.

**Comment by arennuit** [comment]
@nopitjahjadi v3.1.2 missed the ros related packages needed to compile vrep_ros_control_example. vrep_skeleton_msg_and_srv is one example of the missing packages (as far as I can remember there was another ros package missing). This is the reason why I went from v3.1.2 to v3.1.3 and this also means that v3.1.2 will not compile, as you observe.
The release notes for v3.1.3 specifically mention that they added the 2 missing packages: http://www.coppeliarobotics.com/helpFiles/en/versionInfo.htm :
"Added project files for custom ROS plugins for V-REP (i.e. in order to support specific ROS messages of a specific robot). They are located in programming/ros_packages/vrep_plugin_skeleton and programming/ros_packages/vrep_skeleton_msg_and_srv." 
Let me know how it goes.

**Comment by nopitjahjadi** [comment]
so, in instant I should move from 3.1.2 to 3.1.3 in order to use this plugin, right? no ?

**Comment by arennuit** [comment]
Yep ;)

**Comment by JadBatmobile** [comment]
There is no ros_packages directory in the programming folder in the V-REP repository...

**Comment by JadBatmobile** [comment]
Also, using catkin_make, i had to modify CMakeLists.txt file such that:

`target_link_libraries(v_repExtRosControl
    pthread
    dl
    boost_system
   ${catkin_LIBRARIES}
)`
